CHANGELOG.md

```diff
@@ -6,4 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/eth-simple-keyring/
+## [5.0.0]
+### Changed
+- Always return rejected Promise upon failure ([#85](https://github.com/MetaMask/eth-simple-keyring/pull/85))
```

Comment:
Nit: usually we list breaking changes first in each section

Reply:
done [here](https://github.com/MetaMask/eth-simple-keyring/pull/126/commits/9f95eddae32de3aaf10179b468d7f307fe905629)